src/app/share/views/SharedLinksView/SharedView.tsx

```diff
@@ -48,11 +51,25 @@ function copyShareLink(type: string, code: string, token: string) {
 
 export const ITEMS_PER_PAGE = 15;
 
+const removeDuplicates = (list: AdvancedSharedItem[]) => {
```

Comment:
Why are you having to do this?

Reply:
To refresh the last page of pagination when uploading files to a shared folder, as it is requested by pages, if the last page was not complete, it is requested again and you can get up to 14 items that were already in the list. To avoid these duplicates I have applied this solution.

Reply:
Why not increase the offset by the last chunk of the items count received?